pyaerocom/sample_data_access/minimal_dataset.py

```diff
@@ -27,13 +29,17 @@
 
 
 def download_minimal_dataset(
-    file_name: str = TESTDATA_FILE, /, extract_dir_override: str | None = None
+    file_name: str = DEFAULT_TESTDATA_FILE, /, extract_dir_override: str | None = None
 ):
     """Download test_data_file and extracts it.
 
-    :param file_name : The file name to be downloaded.
-    :param extract_dir : An optional folder override to where to extract the file. By
-    default files are extracted into ~/MyPyaerocom
+    Parameters:
+    ===========
+    file_name :
+        The file name to be downloaded.
+    extract_dir :
+        An optional folder override to where to extract the file. By
+        default files are extracted into `~/MyPyaerocom`
```

Comment:
Wrong formatting, use rest or numpy: https://gist.github.com/nipunsadvilkar/fec9d2a40f9c83ea7fd97be59261c400 (in vscode, I like to use https://marketplace.visualstudio.com/items?itemName=njpwerner.autodocstring )
```python
"""
    Download test_data_file and extracts it.

    :param file_name: The file name to be downloaded.
    :param extract_dir: An optional folder override to where to extract the file. By
    default files are extracted into ~/MyPyaerocom
"""
```
should work.

Reply:
Still wrong formatting of parameters at https://pyaerocom--1281.org.readthedocs.build/en/1281/api.html#module-pyaerocom.sample_data_access.minimal_dataset after recents `fix:formatting`: 
![image](https://github.com/user-attachments/assets/f66178d8-dd5a-4e24-8005-f7805a465f1e)

should look like:
![image](https://github.com/user-attachments/assets/338dcddf-a87f-4ea5-9c96-e5be831ed99d)